### PR TITLE
 create and update mutations added

### PR DESCRIPTION
**Pull request by bikov**

### CODE REVIEW

**Review by osher-sade — APPROVED** [review]

File: src/schema/resolvers/mutation-resolvers.ts [hunk 1]
```diff
@@ -1,16 +1,35 @@
...
+                createBook: createBookResolver,
+                updateBook: updateBookResolver,
             },
         };
     }
 }
+
+const createBookResolver = async (parent: object | null, { book }: { book: Book }, context: PolarisContext) => {
+    const { realityId } = context.headers;
+    if (typeof realityId !== 'number') {
```
Comment (osher-sade) [inline src/schema/resolvers/mutation-resolvers.ts]: why aren't you using isNan()?

File: src/schema/resolvers/mutation-resolvers.ts [hunk 2]
```diff
@@ -1,16 +1,35 @@
...
+        throw new UserInputError('please provide reality-id header as number');
+    } else {
+        return await BookModelPerReality(context).create(book);
+    }
+};
+
+const updateBookResolver = async (parent: object | null, { bookId, update }: { bookId: number, update: Partial<Book> }, context: PolarisContext) => {
+    const { realityId } = context.headers;
+    if (typeof realityId !== 'number') {
```
Comment (osher-sade) [inline src/schema/resolvers/mutation-resolvers.ts]: why aren't you using isNan()?

File: package.json [hunk 3]
```diff
@@ -4,14 +4,15 @@
   "description": "example of graphql server using polaris engine",
   "main": "src/index.js",
   "dependencies": {
-    "@enigmatis/mongo-driver": "^1.4.0",
-    "@enigmatis/polaris": "^2.2.1",
+    "@enigmatis/mongo-driver": "file:../mongo-driver",
```
Comment (yarinvak) [inline package.json]: whats that?

File: src/schema/resolvers/mutation-resolvers.ts [hunk 4]
```diff
@@ -1,16 +1,35 @@
...
-                    return { id: book.id, title: book.title, author: book.author };
-                },
+                createBook: createBookResolver,
+                updateBook: updateBookResolver,
             },
         };
     }
 }
+
+const createBookResolver = async (parent: object | null, { book }: { book: Book }, context: PolarisContext) => {
```
Comment (yarinvak) [inline src/schema/resolvers/mutation-resolvers.ts]: Please split the resolvers into separate files

File: src/schema/resolvers/query-resolvers.ts [hunk 5]
```diff
@@ -1,20 +1,28 @@
...
-                        includeOperational,
-                    }: { realityId: number; includeOperational: boolean },
-                ) => BookModelPerReality(realityId).find({}),
+                books: bookQueryResolver,
             },
         };
     }
 }
+
+const bookQueryResolver = async (
```
Comment (yarinvak) [inline src/schema/resolvers/query-resolvers.ts]: split resolvers into separate files

File: package.json (hunk 3 above)
Comment (bikov) [reply package.json]: this is how we install local package in develop time, i'll change that after new version of polaris itself will be deployed.

**Pull request merged**